x-pack/plugins/enterprise_search/public/applications/shared/layout/side_nav.test.tsx

```diff
@@ -79,6 +79,30 @@ describe('SideNavLink', () => {
     expect(wrapper.find('.enterpriseSearchNavLinks__item--isActive')).toHaveLength(1);
   });
 
+  it("won't set an active class when route is a subroute of 'to'", () => {
```

Comment:
[not a blocker] What are your thoughts on wrapping these two new tests in a `describe('shouldShowActiveForSubroutes' () => {` block for organization and swapping the order (testing the positive first and then confirming the negative)? no worries if not, it's super minor

Reply:
Haha. I wanted to, but it was late in the day and and I was trying to get out the door so I thought I'd roll the dice 🎲